config/application.rb

```diff
@@ -29,6 +29,7 @@ class Application < Rails::Application
     config.autoload_paths << Rails.root.join('lib')
 
     Insights::API::Common::Logging.activate(config)
-    Insights::API::Common::Metrics.activate(config, "topological_inventory_ingress_api")
+    require 'topological_inventory/ingress_api/metrics'
+    Insights::API::Common::Metrics.activate(config, "topological_inventory_ingress_api", ::TopologicalInventory::IngressApi::Metrics::CUSTOM_METRICS)
```

Comment:
I wonder if you changed `.activate` to `.activate_custom` and pass the naming hashs' without the need for a separate class as an array?
```
 Insights::API::Common::Metrics.activate(config, "topological_inventory_ingress_api", [custom_metric_hash_1, custom_metric_hash_2])
```

Reply:
Updated this to just have the custom metrics hash inline - I had debated doing it this way initially. Either this way _or_ we could have a constant that contains the array so the line would look like:
`Insights::API::Common::Metrics.activate(config, "topological_inventory_ingress_api", :custom_metrics => Some::Constant::METRICS)`

I'm open to either, but I definitely like having the `:custom_metrics` key right in the line so it's more obvious what is getting passed in.

Reply:
the current way with `:custom_metrics` key looks good for me.